.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing
+So you want to contribute to this repository?
+
+When contributing to this repository, please first discuss the change you wish to make via issue with the owners of this repository before making a change.
+
+## Coding Standards
+Try to make it clean and easy to read with comments as necessary. A `README.md` is required for every problem solution. Please follow the standards put forth in the Templates folder.
```

Comment:
"Try to make it clean" might be more clear if you said "try to make your code clean".